doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc

```diff
@@ -0,0 +1,28 @@
+[id='configuring-emails-in-task-notification-proc']
+
+= Configuring emails in task notification
+
+You can directly add any email addresses to send task notification emails to the users and groups.
```

Comment:
I would reword this. Prior to the change you could send notifications to users and/or groups. Now with the change you can send notifications to users and/or groups and/or emails addresses

Reply:
Okay! I will restructure the sentence.